static/examples/audio.html

```diff
@@ -0,0 +1,188 @@
+              source = audioCtx.createMediaStreamSource(stream);
+              source.connect(distortion);
+              distortion.connect(biquadFilter);
+              biquadFilter.connect(gainNode);
+              gainNode.connect(delay);
+              delay.connect(audioCtx.destination);
+
+              range.oninput = function() {
+                gainNode.gain.value = range.value;
+              }
+              // Delay time needs to between 0, 1 but the slider only returns integers
+              delayrange.oninput = function() {
+                delay.delayTime.value = delayrange.value / 10;
+              }
+              voiceChange();
+            })
+            .catch( function(err) { console.log('The following gUM error occured: ' + err);})
+        } else {
+          console.log('getUserMedia not supported on your browser!');
+        }
+
+        function voiceChange() {
+
+          distortion.oversample = '4x';
+          biquadFilter.gain.setTargetAtTime(0, audioCtx.currentTime, 0)
+
+          let voiceSetting = voiceSelect.value;
+
+            biquadFilter.disconnect(0);
+            biquadFilter.connect(gainNode);
+
+          if(voiceSetting == "distortion") {
+            distortion.curve = makeDistortionCurve(400);
+            biquadFilter.disconnect(0);
+            biquadFilter.connect(gainNode);
+          } else if(voiceSetting == "biquad") {
+            biquadFilter.type = "lowshelf";
+            biquadFilter.frequency.setTargetAtTime(1000, audioCtx.currentTime, 0)
+            biquadFilter.gain.setTargetAtTime(25, audioCtx.currentTime, 0)
+          }
+        }
+
+        const mute = document.getElementById('mute');
+        voiceSelect.onchange = function() {
+          voiceChange();
+        };
+
+        mute.onclick = voiceMute;
+
+        function voiceMute() {
+          if(mute.id === "") {
+            gainNode.gain.value = 0;
+            mute.id = "activated";
+            mute.innerHTML = "Unmute";
+          } else {
+            gainNode.gain.value = 1;
+            mute.id = "";
+            mute.innerHTML = "Mute";
+          }
+        }
+      }
+    </script>
+  </head>
+  <body>
+    <input type="button" value="Start Microphone" onclick="start()"/>
+    <canvas class="visualizer" width="640" height="100"></canvas> 
+    <form class="controls">
+      <div>
+        <label for="voice">Voice setting</label>
+        <select id="voice" name="voice">
+          <option value="distortion">Distortion</option>
+          <option value="biquad">Bass Boost (BiQuad)</option>
+          <option value="off" selected>Off</option>
+        </select>
+      </div>
+      <label for="range">Microphone Gain</label>
+      <div>
+        <input id="range" type="range" min="1" max="40">
+      </div>
+      <label for="delay">Increase Input Delay</label>
+      <div>
+        <input id="delayrange" type="range" min="0" max="10">
+      </div>
+      <div>
+       <div class="mute" id="mute">
```

Comment:
This should be a `<button>` because it is clickable.

There's a similar button for 'Start Microphone' at the top

Your `<input type="button"...` elements should be `<button type="button"...` elements.

Here's a few opinions if you wonder about `<button>` vs `<input>`:
- Short version: https://html.com/attributes/button-type/
- StackOverflow rathole: https://stackoverflow.com/questions/469059/button-vs-input-type-button-which-to-use

The first of these buttons specifies the click handler in the HTML, the second one adds it in the JS. I think that both buttons should use the HTML `onclick` attribute.

---

#### Non-actionable comment

Two-state buttons (e.g., Mute/Unmute) can often be replaced with checkboxes, as that is sometimes less ambiguous in a UI zone (to me), and potentially more *accessible*. I like a checkbox named `Muted`, and if it is checked, that means I'm muted. A non-checkbox named `Mute` or `Unmute` requires my brain to *invert* the word to understand my current state (Oh, it says 'Mute', so I must be unmuted right now). Of course, if you use a checkbox, then you don't use a `<checkbox>` element (there is no such thing), you will use `<input type="checkbox"`. I'm TOTALLY fine with not using checkboxes in this particular case, though.